modules/transport-netty4/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -17,6 +17,11 @@
  * under the License.
  */
 
+grant {
+    // needed for Netty since they use reflection in ThrowableUtil :(
+    permission java.lang.RuntimePermission "accessDeclaredMembers";
```

Comment:
Can we open an issue in netty and link here that netty should work gracefully if it does not have this permission?

Reply:
Opened https://github.com/netty/netty/issues/7614 and added in the comment of this section